Source/App/CMakeLists.txt

```
@@ -45,4 +46,4 @@ if (MSVC OR MSYS OR MINGW OR WIN32)
                            SvtHevcEnc)
 endif()
 
-install(TARGETS SvtHevcEncApp RUNTIME DESTINATION bin)
\ No newline at end of file
+install(TARGETS SvtHevcEncApp RUNTIME DESTINATION bin)
```

```suggestion
install(TARGETS SvtHevcEncApp RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```
alternatively change it to bindir
while adding
``` cmake
if(NOT CMAKE_INSTALL_BINDIR)
    set(CMAKE_INSTALL_BINDIR bin)
endif()
```
somewhere else